lib/file-operations/command-create.js

```diff
@@ -9,7 +9,7 @@ const commandCreate = (alias, command) => {
   try {
 
     // Creating squash config directory
-    directoryCreate();
+    // directoryCreate(); commented out as it is duplicate call
```

Comment:
Do not comment it out.
You can remove this code directory.
We do not want to keep the unused code in the codebase.
Also, if we are not using directoryCreate() function you have to remove the import also, so the lints will not fail

Reply:
Removed the call as advised.